### PR TITLE
Implement react-ga4

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Implements Google analytics 4

### CODE REVIEW

**Review by ramirotw — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @nenadV91 , could you please clarify what I need to test here?

**Comment by nenadV91** [comment]
@elena-zh If you have access to the GA console you could test if Page View events are getting there but if not that nothing.

**Comment by elena-zh** [comment]
No, I don't have a GA console.

File: src/apps/explorer/pages/AppData/index.tsx [hunk 1]
```diff
@@ -69,6 +71,9 @@ const AppDataPage: React.FC = () => {
 
   return (
     <Wrapper>
+      <Helmet>
+        <title>AppData Details - {APP_TITLE}</title>
+      </Helmet>
```
Comment (alfetopito) [inline src/apps/explorer/pages/AppData/index.tsx:76]: Same comment as https://github.com/cowprotocol/cowswap/pull/1097#discussion_r980300973

**Pull request merged**